### ISSUE TITLE
Coinbase login error when using autofill: "Invalid email address" (when it is 100% valid)

### ISSUE THREAD

**Issue by fortXIV** [bitwarden/clients#7120]
### Steps To Reproduce

1. Go to '[Coinbase Sign In](https://login.coinbase.com/signin)'
2. Click on 'browser extension and autofil, it will then fill in your email address and you proceed you to the next page. After that you will have to autofill again (for your password this time), but when that happens you get this error: "Invalid email address" Keep in mind the email is valid and works fine when you manually copy and paste everything'

### Expected Result

Autofills email and password without giving you the "Invalid email address" error

### Actual Result

Gives you this error: "Invalid email address"

### Screenshots or Videos

https://i.imgur.com/CSyawXD.png

### Additional Context

Not really any, just trying to login and buy some crypto.

### Operating System

Windows

### Operating System Version

11 pro, 10.0.22621 Build 22621

### Web Browser

Brave

### Browser Version

1.60.125 Chromium: 119.0.6045.199

### Build Version

2023.10.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by fortXIV** [comment]
edited a new link that actually shows the issue, not blurs it out, lmao.

**Comment by Krychaz** [comment]
Hello there, 

Are you sure there is no extra blank spaces saved within Bitwarden?

**Comment by fortXIV** [comment]
> Hello there,
> 
> Are you sure there is no extra blank spaces saved within Bitwarden?

Yep im certain there no extra blank spaces. It works perfectly when i copy and paste directly from bitwarden + i checked using a site to see if that was the issue - no blank spaces for the email or password. Also tested in firefox, didnt work there either.

**Comment by michal-rybinski** [comment]
Operating System
MacOS

Operating System Version
14.2.1

Web Browser
Firefox 121

Just hit exactly same problem today, same results with Bitwarden but was successfull with autofill from 1Password

**Comment by fort116** [comment]
Bump, most likely something breaking autofill with this specific site (i assume)

**Comment by Amoeba00** [comment]
Yup - still an issue with autofill password on this site.  The autofill icon does appear for the password - but depending on where you click - nothing happens or it kicks you back to the email field.  Selecting the auto-fill login from the extension menu or the right-click context menu - does nothing.  However, copy/paste directly into password field works and the passkey confirmation works as well.

**Comment by fortXIV** [comment]
Bump - this is still an issue, tried to login yesterday and it still persists.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
